File: java/src/main/java/me/anviks/codewars/solutions/_4kyu/parseint_reloaded/SolutionTest.java
/*
 * https://www.codewars.com/kata/525c7c5ab6aecef16e0001a5
 */

package me.anviks.codewars.solutions._4kyu.parseint_reloaded;

import org.junit.Test;
import static org.junit.Assert.assertEquals;
import org.junit.runners.JUnit4;


public class SolutionTest {
    
    @Test
    public void fixedTests() {
        assertEquals(1 , Parser.parseInt("one"));
        assertEquals(20 , Parser.parseInt("twenty"));
        assertEquals(246 , Parser.parseInt("two hundred forty-six"));
        assertEquals(783919, Parser.parseInt("seven hundred eighty-three thousand nine hundred and nineteen"));
    }
}

File: kotlin/src/main/kotlin/me/anviks/codewars/solutions/_6kyu/bouncing_balls/SolutionTest.kt
/*
 * https://www.codewars.com/kata/5544c7a5cb454edb3c000047
 */

package me.anviks.codewars.solutions._6kyu.bouncing_balls

import org.junit.Assert.*
import org.junit.Test

class SolutionTest {
    @Test
    fun test1() {
        assertEquals(3, me.anviks.codewars.solutions._6kyu.bouncing_balls.bouncingBall(3.0, 0.66, 1.5))
    }

    @Test
    fun test2() {
        assertEquals(15, me.anviks.codewars.solutions._6kyu.bouncing_balls.bouncingBall(30.0, 0.66, 1.5))
    }
}
